config/seckit.settings.yml

```
@@ -0,0 +1,56 @@
+seckit_xss:
+  csp:
+    checkbox: true
+    vendor-prefix:
+      x: false
+      webkit: false
+    report-only: true
+    default-src: ''
+    script-src: '''self'' ''unsafe-inline'' https://fonts.googleapis.com https://www.gstatic.com https://www.google.com https://www.google-analytics.com https://cdn.jsdelivr.net'
+    object-src: ''
+    img-src: '''self'' data:'
+    media-src: ''
+    frame-src: ''
+    frame-ancestors: ''
+    child-src: ''
+    font-src: 'https://fonts.gstatic.com'
+    connect-src: '''self'' https://www.google-analytics.com https://stats.g.doubleclick.net'
+    report-uri: /report-csp-violation
+    upgrade-req: false
+    policy-uri: ''
+    style-src: ''
```

```suggestion
    style-src: '''self'' https://fonts.googleapis.com'
```

I think we need the following modification to display Google Fonts. I used HID CSP as my reference.